wordpress-mobile/gutenberg-mobile
# Fix undo/redo on new Para/Heading blocks

This PR fixes a problem with undo/redo not working on new (empty) blocks after writing some text.
Fixes #360 

GB side PR: https://github.com/WordPress/gutenberg/pull/12815

Steps to test:
- Create a new post
- Start a paragraph
- Type hello
- Tap undo: it should go back to an empty paragraph
- Tap undo: paragraph removed
- Tap redo: empty paragraph restored
- Tap redo: "hello" restored

Note: The cursor is not restored to the correct position at the last step. we have another PR open for this: https://github.com/wordpress-mobile/gutenberg-mobile/issues/309
